.gitignore

```diff
@@ -1,10 +1,11 @@
-.idea/
 findfolders
 listrecordings
 upload
+!*/
```

Comment:
Any recollection what this was added for?

It confused me for a bit because it was causing my global gitignore `.vscode` entry to get overridden.

```
~/elastic/zat master*                                                                             
❯ git check-ignore .vscode --verbose
.gitignore:4:!*/	.vscode
```

Reply:
Interesting, I didn't anticipate that side effect.  I don't recall why we did it that way but would be fine to remove it

Reply:
Thanks for checking! I removed it in https://github.com/graphaelli/zat/pull/25

Also sent you a bunch of little cleanup PRs. Happy Holidays! 🎄